scripts/cms/README.md

```diff
@@ -0,0 +1,49 @@
+Description:
+This script goes through all video files in a specific google drive folder and connects with
+ndn-mongo-fileserver, if there is any change from google drive folder e.g. update, delete, insert, this script will handle it.
+
+Some prerequisites for this script: 
+1.Install gdrive from this link:
+  https://github.com/prasmussen/gdrive
+  Choose a proper way to install gdrive from the link.
+2.Install google authenticate module and enable google drive api from this link:
+  https://developers.google.com/drive/api/v3/quickstart/python
+  After you are done with step 1 in the link, you will get a json file, and please name it "credapi.json", and also make sure it is in the cms folder.
+  Note: Step 2 in the link includes commands for all python modules you need to install, please follow those commands.
+  Step 3 in the link will be Run the sample code, please you can run this sample code before you run this script.
+3.Install dateutil module: pip install python-dateutil.
+
+How to run this script:
+This script needs two options when you want to run it.
+-v means version (you should choose from either info or debug)
+-n means the name of folder (the name of folder in google drive)
+For example if you want to run it in info mode with folder ndn you should apply python ndn_script.py -v info -n ndn
+You can also use python ndn_script.py -h or python ndn_script.py --help to check usage.
+
```

Comment:
Add a section for Troubleshooting.

Add this under this new section:

If there is any mismatch between `urllib3` and `chardet` modules run the following command (you might need to run in `sudo` mode) to resolve the issue:

    `$ pip install requests`